README.md

```diff
@@ -217,6 +221,22 @@ If you want to update a specific form there is a function
 (inf-clojure-update-feature 'clojure 'completion "(incomplete.core/completions \"%s\")")
 ```
 
+### `clojure-ts-mode` support
+
+To use `inf-clojure` with `clojure-ts-mode`, you can adjust the
+following variable:
+
+```emacs-lisp
+(setopt inf-clojure-source-modes '(clojure-ts-mode clojure-mode))
```

Comment:
Shouldn't we just make this the default?

Reply:
Please let me know if I should proceed with changing the default.

Reply:
Yeah, you can proceed.

Reply:
Done. Also adjusted the README.